tech-committee-charter.md

```diff
@@ -100,7 +100,13 @@ The TC follows a [Consensus Seeking](https://en.wikipedia.org/wiki/Consensus-see
 
 If an agenda item cannot reach a consensus a TC member can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be seconded by a majority of the TC or else the discussion will continue.
 
-For all votes, a simple majority of all TC members for, or against, the issue wins. A TC member may choose to participate in any vote through abstention.
+For all votes, a simple majority of all TC members for, or against, the issue wins. A TC member may choose to participate in any vote through abstention. TC votes may be done synchronously or asynchronously. Asynchronous votes remain open for a period of time agreed upon by the TC when the issue is put to a vote. Aggregated TC vote results are shared publicly but the vote selections of individual members are not shared. The TC decides whether vote selections by members are anonymous or shared among members when the issue is put to a vote.
+
+### Requesting a TC decision
+
+When a project issue fails to reach consensus, participants may request that the TC make a decision. 
```

Comment:
Are we okay with any participants or we want to scope this to project leads for now? I'm considering the situation where the SIG reached consensus among majority of folks while a single participant wanted to escalate to the TC.

Reply:
I think minimally we should scope it to otel community members. Further narrowing to project leads seems helpful if there's a simple definition for what a project lead is. Could say that it has to be 1 or 2 maintainers / approvers.